bql/bql_box_test.go

```diff
@@ -325,7 +331,10 @@ func TestBQLBoxEmitterParams(t *testing.T) {
 		})
 	})
 
-	Convey("Given a BQL statement with a SAMPLE clause", t, func() {
+	// This test's outcome should check after "box" emit 40 tuples.
```

Comment:
`Because SAMPLE is an unofficial feature and occasional failures of this test disturb CI, this test is also disabled.`